package.json

```diff
@@ -17,19 +17,15 @@
     "build": "yarn workspaces foreach -pt run build",
     "lint": "eslint . --ext .js,.cjs,.mjs,.ts,.cts,.mts --fix --ignore-path .gitignore",
     "lint:ci": "eslint . --ext .js,.cjs,.mjs,.ts,.cts,.mts --ignore-path .gitignore",
-    "test": "ava",
-    "coverage": "c8 ava",
+    "test": "NODE_OPTIONS='--loader=tsx --no-warnings' ava",
```

Comment:
Is there a reason these were moved from the `ava` configuration?

I usually use `yarn ava src/.../file.spec.ts` to test specific files, but I guess it doesn't hurt to do `yarn test src/...file.spec.ts` instead 🤔

Reply:
I made this change because builds were failing on node 18 and 20, due to ava being unable to load the .ts files.

I found this solution in some github issue i cant find again. When i tried to find it again i found this instead https://github.com/avajs/ava/blob/main/docs/recipes/typescript.md#nodejs-20

As far as i can see, calling `yarn test` and `yarn coverage` works as expected.

Reply:
Ah right, Zeepkist and all of my previous TS refactors happened under Node 21 which no longer has that limitation 😅

Will close this as resolved for compatibility with 18/20